.github/workflows/code_formatting.yml

```
@@ -1,14 +1,23 @@
 name: Check code formatting
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - **/*.c
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 8: did not find expected alphabetic or numeric character

(syntax-check)